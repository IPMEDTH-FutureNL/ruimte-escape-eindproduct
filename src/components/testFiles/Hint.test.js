import renderer from 'react-test-renderer';
import Hint from "../Hint";

it('Check if hint is rendered with the right type', () => {
    const component = renderer.create(
        <Hint type='dragAndDropHints'/>,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})