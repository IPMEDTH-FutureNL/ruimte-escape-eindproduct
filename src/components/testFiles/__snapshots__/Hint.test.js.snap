// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Check if hint is rendered with the right type 1`] = `
Array [
  <button
    aria-describedby="popup-1"
    className="hint-button"
    onClick={[Function]}
  >
     hints
  </button>,
  <div
    className="punten"
  >
    Punten: 
  </div>,
]
`;
